tests/Domain/Package/PackageManagerTest.php

```diff
@@ -171,7 +188,8 @@ public function testConstructionAndGetters(): void {
     self::assertEquals('drupal/module2', $package->getPackageName(), 'Got package by name.');
     self::assertEquals(self::EXPECTED_DEPENDENCY_LIST, $actual_dependency_list, 'Set/got all dependencies.');
     self::assertEquals(TRUE, $package->isCompanyPackage(), 'Got a company package.');
-    self::assertEquals(FALSE, $manager->get('drupal/dependency1')->isCompanyPackage(), 'Got a third party dependency.');
+    self::assertEquals(FALSE, $manager->get('drupal/dependency1')
+      ->isCompanyPackage(), 'Got a third party dependency.');
```

Comment:
Non-meaningful (whitespace only) change.

```suggestion
    self::assertEquals(FALSE, $manager->get('drupal/dependency1')->isCompanyPackage(), 'Got a third party dependency.');
```